enterprise/backend/test/metabase_enterprise/workspaces/test_util.clj

```diff
@@ -28,6 +30,17 @@
                                                 :model/WorkspaceDependency]
                           (tests)))))
 
+(derive :model/Workspace :model/WorkspaceCleanUpInTest)
```

Comment:
What's your concern with just adding this in the app?

I guess the fact that it's synchronous? You could still have the hook fire off a task.

Reply:
> I guess the fact that it's synchronous?

yea, I find the idea of doing side affects other than toucan related operations is dangerous here.